features/storage/csi_clone.feature

```diff
@@ -41,4 +41,96 @@ Feature: CSI clone testing related feature
     Then the output should contain "clone test"
     When I execute on the pod:
       | sh | -c | echo "clone test" > /mnt/local/testfile_2nd |
-    Then the step should succeed
\ No newline at end of file
+    Then the step should succeed
+
+
+  # @author wduan@redhat.com
+  # @case_id OCP-27689
+  Scenario: [Cinder CSI Clone] Clone a pvc with capacity greater than original pvc
+    Given I have a project
+    # Create mypvc-ori with 1Gi size
+    Given I obtain test data file "storage/misc/pvc.json"
+    When I create a dynamic pvc from "pvc.json" replacing paths:
+      | ["metadata"]["name"]                         | mypvc-ori    |
+      | ["spec"]["storageClassName"]                 | standard-csi |
+      | ["spec"]["resources"]["requests"]["storage"] | 1Gi          |
+    Then the step should succeed
+    Given I obtain test data file "storage/misc/pod.yaml"
+    When I run oc create over "pod.yaml" replacing paths:
+      | ["metadata"]["name"]                                         | mypod-ori  |
+      | ["spec"]["volumes"][0]["persistentVolumeClaim"]["claimName"] | mypvc-ori  |
+      | ["spec"]["containers"][0]["volumeMounts"][0]["mountPath"]    | /mnt/local |
+    Then the step should succeed
+    Given the pod named "mypod-ori" becomes ready
+    When I execute on the pod:
+      | sh | -c | echo "clone test" > /mnt/local/testfile |
+    Then the step should succeed
+    And I execute on the pod:
+      | sh | -c | sync -f /mnt/local/testfile |
+    Then the step should succeed
+
+    # Clone mypvc-ori with 3Gi size
+    Given I obtain test data file "storage/csi/pvc-clone.yaml"
+    When I create a dynamic pvc from "pvc-clone.yaml" replacing paths:
+      | ["metadata"]["name"]                         | mypvc-clone  |
+      | ["spec"]["storageClassName"]                 | standard-csi |
+      | ["spec"]["resources"]["requests"]["storage"] | 3Gi          |
```

Comment:
How about using a smaller size, like 2Gi, 1.5Gi? To save resources

Reply:
Updated to 2Gi